eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -3471,6 +3472,67 @@ public function testMoveSubtreeKeepsContentHiddenOnChildren(): void
         }
     }
 
+    /**
+     * Test validating whether content that is being moved is still allowed to be moved when one of its locations
+     * is inaccessible by a current user, however, when moved location is accessible.
+     *
+     * @covers \eZ\Publish\API\Repository\LocationService::moveSubtree
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
```

Comment:
For test purpose it's enough to:
```suggestion
     * @throws \eZ\Publish\API\Repository\Exceptions\Exception
```